[2022-01-25 14:55:30,172] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2022-01-25T14:55:03.016519+00:00 [queued]>
[2022-01-25 14:55:30,188] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2022-01-25T14:55:03.016519+00:00 [queued]>
[2022-01-25 14:55:30,189] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 14:55:30,189] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 14:55:30,190] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 14:55:30,206] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): apply_model> on 2022-01-25T14:55:03.016519+00:00
[2022-01-25 14:55:30,214] {standard_task_runner.py:52} INFO - Started process 2741 to run task
[2022-01-25 14:55:30,221] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'apply_model', '2022-01-25T14:55:03.016519+00:00', '--job-id', '183', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpldo937i2', '--error-file', '/tmp/tmphhw42ca1']
[2022-01-25 14:55:30,229] {standard_task_runner.py:77} INFO - Job 183: Subtask apply_model
[2022-01-25 14:55:30,284] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.apply_model 2022-01-25T14:55:03.016519+00:00 [running]> on host 42ecca77b6ba
[2022-01-25 14:55:30,335] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=apply_model
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T14:55:03.016519+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-25T14:55:03.016519+00:00
[2022-01-25 14:55:52,371] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 14:55:52,393] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=apply_model, execution_date=20220125T145503, start_date=20220125T145530, end_date=20220125T145552
[2022-01-25 14:55:52,455] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 14:55:52,491] {local_task_job.py:146} INFO - Task exited with return code 0
